### PR TITLE
Solve circular import in gammapy.visualization

### PR DESCRIPTION
**Pull request by registerrier**
**Description**

This pull request solves issue #4065 . 
Should we create tests for global module imports?

**Dear reviewer**

### CODE REVIEW

**Review by adonath — CHANGES_REQUESTED** [review]
Thanks @registerrier! I think we should resolve the error the other way around:
- All modules should be able to import from `gammapy.visualization`
- Use delayed imports in `gammapy.visualization`

What do you think?

File: gammapy/maps/region/geom.py [hunk 1]
```diff
@@ -794,6 +793,7 @@ def plot_region(self, ax=None, kwargs_point=None, path_effect=None, **kwargs):
         ax : `~astropy.visualization.WCSAxes`
             Axes to plot on.
         """
+        from gammapy.visualization.utils import ARTIST_TO_LINE_PROPERTIES
```
Comment (adonath) [inline gammapy/maps/region/geom.py:796]: I thunk this can remain a non-delayed import...

Comment (registerrier) [reply gammapy/maps/region/geom.py]: Done

**Review by adonath — APPROVED** [review]
Thanks @registerrier, no further comments from my side.

**Pull request merged**